.github/workflows/docker-publish-release.yaml

```diff
@@ -0,0 +1,80 @@
+name: docker-publish-release
+
+on:
+  push:
+    branches:
+      - pschork/semver
+    tags:
+      - v*
+  pull_request:
+  workflow_dispatch:
+    inputs:
+      force:
+        description: "Force untagged release (expert mode)"
```

Comment:
What's the usecase for pushing an untagged release?

Reply:
Allows someone to release a feature branch for debugging. Running the workflow manually should be rare.